cmd/gardenlet/app/bootstrap.go

```diff
@@ -44,15 +47,18 @@ func bootstrapKubeconfig(
 		return nil, "", "", err
 	}
 
-	if len(gardenKubeconfig) > 0 {
-		logger.Info("Found kubeconfig generated from bootstrap process. Using it")
-		return gardenKubeconfig, "", "", nil
+	gardenKubeconfigFound := len(gardenKubeconfig) > 0
+	if gardenKubeconfigFound {
+		logger.Info("Found kubeconfig generated from bootstrap process. Fetching boostrap kubeconfig to check for changed kube-apiserver CA.")
+	} else {
+		logger.Info("No kubeconfig from a previous bootstrap found. Starting bootstrap process.")
 	}
 
-	logger.Info("No kubeconfig from a previous bootstrap found. Starting bootstrap process.")
-
 	if config.GardenClientConnection.BootstrapKubeconfig == nil {
-		logger.Warn("Unable to perform kubeconfig bootstrapping. The gardenlet configuration `.gardenClientConnection.bootstrapKubeconfig` is not set")
+		logger.Warn("Unable to fetch boostrap kubeconfig. The gardenlet configuration `.gardenClientConnection.bootstrapKubeconfig` is not set")
```

Comment:
```suggestion
		logger.Warn("Unable to fetch bootstrap kubeconfig. The gardenlet configuration `.gardenClientConnection.bootstrapKubeconfig` is not set")
```

Reply:
This log should probably only be written if the garden kubeconfig was not found?